pages/workspace/tracking-plan/exporting.mdx

```diff
@@ -6,10 +6,18 @@ title: 'Exporting'
 
 import Link from '../../../components/Link';
 
-Go to **Settings** :gear: > **Export** to export your full or partial tracking plan in a CSV or JSON format. (In beta, request access via support@avo.sh).
+<Link passHref href="/workspace/tracking-plan/publishing">Publishing integrations</Link> can be used for exporting your tracking plan, either one of, or a recurring publishing export using auto-publishing.
```

Comment:
These `<Link>`s don't seem to have any styling applied to them. I think that it's because they're missing the `<a>` element inside them.
```suggestion
<Link passHref href="/workspace/tracking-plan/publishing"><a>Publishing integrations</a></Link> can be used for exporting your tracking plan, either one of, or a recurring publishing export using auto-publishing.
```